alvr/dashboard/src/main.rs

```diff
@@ -22,14 +22,25 @@ use dashboard::Dashboard;
 #[cfg(not(target_arch = "wasm32"))]
 fn main() {
     use alvr_common::ALVR_VERSION;
+    use alvr_filesystem as afs;
     use eframe::{
         egui::{IconData, ViewportBuilder},
         NativeOptions,
     };
     use ico::IconDir;
-    use std::{env, fs};
+    use std::{env, ffi::OsStr, fs};
     use std::{io::Cursor, sync::mpsc};
 
+    // Kill any other dashboard instance
+    let self_path = std::env::current_exe().unwrap();
+    for proc in sysinfo::System::new_all().processes_by_name(OsStr::new(&afs::dashboard_fname())) {
+        if let Some(other_path) = proc.exe() {
+            if other_path == self_path {
+                proc.kill();
+            }
```

Comment:
How does this not also kill our current dashboard instance?

Reply:
it should have been `other_path != self_path`